rmgpy/rmg/model.py

```diff
@@ -995,7 +1004,80 @@ def addSpeciesToEdge(self, spec):
         Add a species `spec` to the reaction model edge.
         """
         self.edge.species.append(spec)
-
+    
+    def setThermodynamicFilteringParameters(self,Tmax, toleranceThermoKeepSpeciesInEdge,minCoreSizeForPrune,maximumEdgeSpecies,reactionSystems):
+        """
+        sets parameters for thermodynamic filtering based on the current core
+        """
+        self.Tmax = Tmax
+        Gs = [spc.thermo.getFreeEnergy(Tmax) for spc in self.core.species]
+        self.Gmax = max(Gs)
+        self.Gmin = min(Gs)
+        
+        self.Gfmax = toleranceThermoKeepSpeciesInEdge*(self.Gmax-self.Gmin)+self.Gmax
+        self.toleranceThermoKeepSpeciesInEdge = toleranceThermoKeepSpeciesInEdge
+        self.minCoreSizeForPrune = minCoreSizeForPrune
+        self.reactionSystems = reactionSystems
+        self.maximumEdgeSpecies = maximumEdgeSpecies
+        
+    def thermoFilterDown(self,maximumEdgeSpecies,minSpeciesExistIterationsForPrune=0):
+        """
+        removes species from the edge based on their Gibbs energy until maximumEdgeSpecies
+        is reached under the constraint that all removed species are older than
+        minSpeciesExistIterationsForPrune iterations
```

Comment:
This isn't a new parameter, it's from older pruning.  Originally I was going to accommodate that parameter for thermodynamic pruning, but I decided that one of the primary advantages of thermodynamic pruning is that you don't have to run a simulation to filter things, which you lose if you have a minIterationsSpeciesExistForPrune.  I left support for it in thermoFilterDown because I can see situations where it might be desirable to implement that feature for thermodynamic pruning, but it isn't currently used.